app/routes/articles/components/Overview.tsx

```diff
@@ -24,16 +29,20 @@ const OverviewItem = ({ article }: { article: ArticleWithAuthorDetails }) => (
     </h2>
 
     <span className={styles.itemInfo}>
-      {article.authors.map((e) => {
-        return (
-          <span key={e.username}>
-            <Link to={`/users/${e.username}`} className={styles.overviewAuthor}>
-              {' '}
-              {e.fullName}
-            </Link>{' '}
-          </span>
-        );
-      })}
+      {article.authors.map(
+        (author) =>
+          typeof author === 'object' && (
```

Comment:
This seems a bit hacky.

Reply:
It was. I've fixed it now so that it shows authors on the public front page.